py-polars/polars/_utils/construction/series.py

```diff
@@ -422,9 +423,10 @@ def pandas_to_pyseries(
             "(e.g. 'int64', 'bool', 'float32' - not 'Int64')"
         )
         raise ImportError(msg)
-    return arrow_to_pyseries(
+    s = arrow_to_pyseries(
         name, plc.pandas_series_to_arrow(values, nan_to_null=nan_to_null)
     )
+    return s if dtype is None else s.cast(dtype, strict=strict)
```

Comment:
rather than doing this here, could we pass `strict` to `arrow_to_pyseries` and do the cast in there?